android/src/com/mapswithme/maps/location/PlatformSocket.java

```diff
@@ -117,17 +150,17 @@ public void close()
   {
     if (mSocket == null)
     {
-      sLogger.d("Socket is already closed or it wasn't opened yet");
+      LOGGER.d("Socket is already closed or it wasn't opened yet\n");
       return;
     }
 
     try
     {
       mSocket.close();
-      sLogger.d("Socket has been closed: ", this);
+      LOGGER.d("Socket has been closed: ", this + "\n");
     } catch (IOException e)
     {
-      sLogger.e("Failed to close socket: ", this, e);
+      LOGGER.e(e, "Failed to close socket: ", this);
```

Comment:
You are right about separate line. But I added additional separate lines to make reading the logs more easily/conveniently - http://clip2net.com/s/3FrlZRW . What do you think about it?